### ISSUE TITLE
Increase readability of rant text on colored background

### ISSUE THREAD

**Issue by valeth** [devRant/devRant#464]
Currently the rant text seems to always be pure white (`#fff`).
This causes issues when the user color is very bright (`#e6c653` for example).

Example from a [rant](https://devrant.com/rants/1530413/tools-i-enjoy-working-with-python-c-visual-studio-code-geany-tools-i-dont-enjoy):
![screenshot_20180702_193750](https://user-images.githubusercontent.com/3198362/42178537-1568805e-7e31-11e8-8a3e-744419737e24.png)

The WCAG contrast ratio for this one is 1.67:1, which is below the recommended ratio.
https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=E6C653

So either the foreground color should be adjusted to a darker color to make it better readable, or the background color needs to be darkened somehow.
